tensorflow/python/ops/image_ops_impl.py

```diff
@@ -638,10 +638,10 @@ def transpose(image, name=None):
     If `image` was 3-D, a 3-D float Tensor of shape
    `[width, height, channels]`
    
-   Usage Example: ```python >> 
-    import tensorflow as tf >> 
-    x = tf.random.normal(shape=(100, 200, 3)) >> 
-    tf.image.transpose(x) `
+   Usage Example: ```python 
+    >>> import tensorflow as tf >> 
```

Comment:
Why the >> at the end of lines?

Also no need to import tensorflow.

Also don't use a random image, use a small constant. Also there are like five other PRs in flight for the same thing I think.

Reply:
About the PR's, these are all student submissions from GCI-2019. Would you like me to check for duplicates and inform those students to redirect elsewhere?